pennylane_quantuminspire2/api/pagination.py

```diff
@@ -1,3 +0,0 @@
-from qiskit_quantuminspire.api.pagination import PageReader
```

Comment:
I'm not really sure why we have this in the code. Convenience for end users? If so, we don't want to offer this from the qi2_shared section?

Reply:
The only reason we had this previously was because we needed it to for the e2e_tests (see [this revision](https://github.com/QuTech-Delft/pennylane-quantuminspire2/blob/c0b588cb281c401c45b2b43684a099746da5c9c0/tests/e2e_test.py#L17-L27) of `e2e_test.py`, namely the `_fetch_team_member_id()` method).

This wasn't really the way to go to begin with (exposing functionality solely for e2e testing), but now that we have the shared github workflows we can drop it altogether. The qiskit-plugin **is** using it from the `qi2_shared` to handle pagination when making calls to the client.